app/models/lookup_keys/lookup_key.rb

```diff
@@ -144,6 +144,12 @@ def overridden?(host)
     host.lookup_values.any? { |lv| lv.lookup_key_id == id }
   end
 
+  def overridden_value(host)
```

Comment:
Changed. Also changed `host` to `obj` since it can be also a host group and in the future possible other objects, and added comment on why we do it by iteration instead of by DB query.